registry/application/handlers/file_handlers.py

```diff
@@ -0,0 +1,26 @@
+from common.constant import StatusCode
+from common.exception_handler import exception_handler
+from common.logger import get_logger
+from common.utils import generate_lambda_response
+from registry.application.access_control.authorization import secured
+from registry.application.services.file_service.file_service import FileService
+from registry.config import SLACK_HOOK, NETWORK_ID
+from registry.constants import Action
+from registry.exceptions import EXCEPTIONS, BadRequestException
+
+logger = get_logger(__name__)
+
+
+@exception_handler(SLACK_HOOK=SLACK_HOOK, NETWORK_ID=NETWORK_ID, logger=logger, EXCEPTIONS=EXCEPTIONS)
+@secured(action=Action.CREATE, org_uuid_path=("queryStringParameters", "org_uuid"),
+         username_path=("requestContext", "authorizer", "claims", "email"))
+def delete(event, context):
+    query_parameters = event["queryStringParameters"]
+
+    if "type" not in query_parameters:
+        raise BadRequestException()
```

Comment:
How are we validating that the service and org details correspond to the user making the call? Is it handled via the roles set up?

Reply:
decorator @secured() takes `org_uuid` `username` and validates whether user is member of the organization or not.

Reply:
`secured(action=Action.CREATE, org_uuid_path=("queryStringParameters", "org_uuid"),
         username_path=("requestContext", "authorizer", "claims", "email"))`